src/main/java/com/ucm/shiro/facturae/v3_0/transformer/FacturaeTransformerToV3_1.java

```diff
@@ -0,0 +1,13 @@
+package com.ucm.shiro.facturae.v3_0.transformer;
+
+import com.ucm.shiro.facturae.agnostic.transformer.AbstractFacturaeTransformer;
+import com.ucm.shiro.facturae.exception.FacturaeTransformException;
+
+public class FacturaeTransformerToV3_1 extends AbstractFacturaeTransformer {
+
+    private static final String FACTURAE_TO_3_1_XSLT_PATH = "xml/schema/facturae/v3_0/Transformer3.0To3.1.xslt";
```

Comment:
Now the transformer transfrom from 3.0 to 3.2.1.

Reply:
With the current approach, the transform is ok